### PR TITLE
typings: number[] and string[] are not valid types for color

### PR DESCRIPTION
**Pull request by KingDGrizzle**
### Changes Proposed in this Pull Request (List new items in CHANGELOG.MD)

-

### Semver Classification

- [x] This PR only includes documentation or non-code changes.
- [x] This PR fixes a bug and does not change the (intended) framework interface.
- [ ] This PR adds methods or properties to the framework interface.
- [ ] This PR removes or renames methods or properties in the framework interface.

### CODE REVIEW

**Comment by kyranet** [comment]
bump, there are requested changes for almost a month.

File: src/lib/util/Colors.js [hunk 1]
```diff
@@ -53,7 +53,7 @@ class Colors {
 	/**
 	 * Apply the style
 	 * @since 0.5.0
-	 * @param {(string|string[])} styles The style or styles to apply
+	 * @param {?(string|string[])} styles The style or styles to apply
```
Comment (bdistin) [inline src/lib/util/Colors.js]: JSDocs doesn't use `?` to denote optional
```suggestion
	 * @param {(string|string[])} [styles] The style or styles to apply
```

File: src/lib/util/Colors.js [hunk 2]
```diff
@@ -74,7 +74,7 @@ class Colors {
 	/**
 	 * Apply the background
 	 * @since 0.5.0
-	 * @param {ColorsFormatType} background The background to apply
+	 * @param {?string} background The background to apply
```
Comment (bdistin) [inline src/lib/util/Colors.js]: JSDocs doesn't use `?` to denote optional
```suggestion
	 * @param {string} [background] The background to apply
```

File: src/lib/util/Colors.js [hunk 3]
```diff
@@ -90,7 +90,7 @@ class Colors {
 	/**
 	 * Apply the text format
 	 * @since 0.5.0
-	 * @param {ColorsFormatType} text The text format to apply
+	 * @param {?string} text The text format to apply
```
Comment (bdistin) [inline src/lib/util/Colors.js]: JSDocs doesn't use `?` to denote optional
```suggestion
	 * @param {string} [text] The text format to apply
```

**Pull request merged**